ingestion/cadet_run_results.py

```diff
@@ -0,0 +1,46 @@
+import os
+from datetime import datetime, timedelta, timezone
+
+import boto3
+import yaml
+
+
+def inject_run_result_paths_into_yaml_template(yaml_path):
+    with open(yaml_path) as f:
+        template = yaml.safe_load(f)
+
+    template["source"]["config"]["run_results_paths"] = get_cadet_run_result_paths()
+
+    # Overwite the original file with updated the run results paths.
+    with open(yaml_path, "w") as f:
+        yaml.dump(template, f, indent=2, sort_keys=False, default_flow_style=False)
+
+
+def get_cadet_run_result_paths(bucket_name="mojap-derived-tables", days=1):
```

Comment:
Is the last 1 days enough? Some of the models are run monthly aren't they?

Reply:
Yeah there are - the files are retained for 14 days only so wouldn't be able to get a month. We could get 14 days one time and then revert to 1?

Reply:
Ahhhh okay I see more now how you envisage this working. We should be totally up to speed in a month from when we introduce this change if all goes well? 1 day will be fine. That figure has a dependency on our pipelines running ok, but people should be able to see when we've updated our records by the `updated 5 hours ago` bit aka "metadata last updated"